
From: Olaf Hering <olh@suse.de>

I'm not sure why these defines and typedefs exists, the driver compiles
fine without it.

Signed-off-by: Olaf Hering <olh@suse.de>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/udf/lowlevel.c |    6 ------
 1 files changed, 6 deletions(-)

diff -puN fs/udf/lowlevel.c~remove-scsi-ioctl-from-udf-lowlevelc fs/udf/lowlevel.c
--- 25/fs/udf/lowlevel.c~remove-scsi-ioctl-from-udf-lowlevelc	2004-10-02 23:27:55.351673184 -0700
+++ 25-akpm/fs/udf/lowlevel.c	2004-10-02 23:27:55.355672576 -0700
@@ -27,12 +27,6 @@
 #include <linux/blkdev.h>
 #include <linux/cdrom.h>
 #include <asm/uaccess.h>
-#include <scsi/scsi.h>
-
-typedef struct scsi_device Scsi_Device;
-typedef struct scsi_cmnd   Scsi_Cmnd;
-
-#include <scsi/scsi_ioctl.h>
 
 #include <linux/udf_fs.h>
 #include "udf_sb.h"
_
